src/useCases/getAccountDetails.ts

```diff
@@ -7,18 +7,7 @@ export type GetAccountDetailsFn = (
 ) => Promise<IAccountHolderDetails>;
 
 export const getAccountDetails =
-  ({
-    getOrCreateAccountId,
-    getAccessToken,
-    accountHolderApiGateway,
-  }: IAppContainer): GetAccountDetailsFn =>
+  ({ getOrCreateAccount }: IAppContainer): GetAccountDetailsFn =>
```

Comment:
if it's not being used i say bin it, but don'y forget to delete `GetAccountDetailsFn` and remove from the appContainer 👍